components/pipeline-service/production/base/update-tekton-config-performance.yaml

```diff
@@ -31,6 +31,14 @@
   path: /spec/pipeline/options/deployments/tekton-operator-proxy-webhook/spec/replicas
   # default pipeline-service setting is 1
   value: 2
+- op: replace
+  path: /spec/pruner
+  value:
+    keep: 10
+    keep-since: 15
```

Comment:
I suggest we run the pruner more often (every 10 or 15 mins for example) and then set the keep-since to 60 (minutes). This way when the watcher pruning is lagging, we won't prune prs which finished soon after x:15.
The schedule will look like `schedule: */15 * * * *` and `keep-since: 60`. WDYT?

Reply:
my yaml editor does not seem to like `*/15 * * * *` but both it and crontab.guru seem to abide `0/15 * * * *`

that also lines up with what Romain used to do i.e. https://github.com/openshift-pipelines/pipeline-service/commit/962f485476bb1c75a8f9f6a743a0cb953e587fbc#diff-7b0f46511f3c76f0deb347cc69126f54ac59102bee06eeee07471cf9fa02b8bdR58

I like your suggestion though ... you good with that slight adjustment @enarha ?

Reply:
update pushed in the interim

Reply:
The config is good, but I think you forgot to regenerate the yamls.

Reply:
Not sure if it matters, but I see all examples putting the value in quotes "*/15 * * * *", so maybe add that as well.

Reply:
both updates (gen deploy and quotes) implemented and pushed @enarha PTAL